src/osd/osd_types.h

```diff
@@ -1167,7 +1167,7 @@ class pool_opts_t {
     if (i == opts.end()) {
       return false;
     }
-    *val = boost::get<T>(i->second);
+    *val = std::get<T>(i->second);
```

Comment:
Just to make sure: that's definitely a get<> and not a get_if<> ?

Reply:
yes, we assign the *value* of the alternative to the pointee of `val`. so it's definitely `std::get<>`.

Reply:
Sure. I was referring to the behaviour when 'not the current alternative' - not the taking of ref.

Reply:
I see. I intended to preserve the existing behavior. If “not the specified alternative”, an exception is thrown. but I don’t think to verify the existing behavior is in the scope of this change, what do you think?

Reply:
I agree